components/QrCodeOverlay.tsx

```diff
@@ -16,16 +16,18 @@ export const QrCodeOverlay: React.FC<QrCodeOverlayProps> = props => {
   return (
     <React.Fragment>
       <Pressable onPress={toggleQrOverlay}>
-        <Row margin="20 0 0 0">
-          <QRCode
-            size={90}
-            value={props.qrCodeDetailes}
-            backgroundColor={Theme.Colors.QRCodeBackgroundColor}
-          />
-        </Row>
+        <View style={Theme.QrCodeStyles.QrView}>
+          <Row margin="0 0 0 0" style={{}}>
```

Comment:
Is there any specific reason to keep an empty-style prop? If not, it's good to remove it.